mcts.py

```diff
@@ -55,7 +55,7 @@ def expand_node(node, model, env):
     value = values[0]
     legal_actions = env.get_legal_actions(node.state)
     for i, action in enumerate(legal_actions):
-        action_prob = action_probs[i]
+        action_prob = action_probs[int(action)]
```

Comment:
What is this doing?

Reply:
This is making sure we index the action_distribution properly.  We want action_probs to have size (env.action_size).  legal_actions will have size (num_legal_actions).  action_probs should be nonzero for the legal actions.  If it was action_probs[i], only the first num_legal_actions would have nonzero probability... but those are not actually the legal actions.

Reply:
Looks like we can get rid of the enumerate. Why do we need to cast as an int?